### DISCUSSION TITLE
Laravel-UrlShortener

### DISCUSSION THREAD

**Discussion by yordadev** [HassanZahirnia/laravel-package-ocean#45] [Package Suggestions]
👋🏼 Hello:

I wanted to suggest my [Laravel URL Shortener package](https://github.com/YorCreative/Laravel-UrlShortener) for inclusion in your list of suggested packages. The package provides URL redirects with optionally protected URL password, Hidden UTM parameter tracing, URL expiration, open limits before expiration, ability to set feature activation dates, and click tracking out of the box for Laravel applications.

I also want to express my gratitude for the inclusion of one of my other packages, [Laravel-Scrubber](https://github.com/YorCreative/Laravel-Scrubber), in your package list. While it was not suggested by me, I appreciate the recognition and support. 🙏🏼

**Comment by HassanZahirnia** [comment]
Hey! Thanks for your suggestion. I see your efforts and dedication, good stuff.

I see your package has not been updated for many months, and doesn't seem be compatible with Laravel 10. I would only add packages that are actively maintained and up to date with latest Laravel versions.

If you attend to updating it more often I'll be happy to add it to the database!

**Reply by yordadev** [reply]
Hey there! Thanks for the comment, I really appreciate the kind words.

I actually just released version 2.1 of my package, and it is fully compatible with Laravel 10. So feel free to add it to the database! Let me know if you have any other questions or concerns.

**Reply by HassanZahirnia** [reply]
I've added the package! 😊
Thanks for keeping your package up to date!
